src/Arcus.Scripting.Storage.Table/Scripts/Set-AzTableStorageEntities.ps1

```diff
@@ -0,0 +1,71 @@
+param(
+    [Parameter(Mandatory = $true)][string] $ResourceGroupName = $(throw "Name of resource group is required"),
+    [Parameter(Mandatory = $true)][string] $StorageAccountName = $(throw "Name of Azure storage account is required"),
+    [Parameter(Mandatory = $true)][string] $TableName = $(throw "Name of Azure table is required"),
+    [Parameter(Mandatory = $true)][string] $ConfigurationFile = $(throw "Path to the configuration file is required")
+)
+
+if ((Test-Path -Path $ConfigurationFile) -eq $false) {
+    throw "Configuration file not found"
+}
+
+# Retrieve Azure storage Account
+Write-Verbose "Retrieving Azure storage account context for Azure storage account '$StorageAccountName' in resource group '$ResourceGroupName'..."
+$storageAccount = Get-AzStorageAccount -ResourceGroupName $ResourceGroupName -Name $StorageAccountName
+if ($storageAccount -eq $null)
+{
+    throw "Retrieving Azure storage account context for Azure storage account '$StorageAccountName' in resource group '$ResourceGroupName' failed."
+}
+$ctx = $storageAccount.Context
+Write-Verbose "Azure storage account context has been retrieved for Azure storage account '$StorageAccountName' in resource group '$ResourceGroupName'"
+
+# Retrieve Azure storage table
+Write-Verbose "Retrieving Azure storage table '$TableName' for Azure storage account '$StorageAccountName' in resource group '$ResourceGroupName'..."
+$storageTable = Get-AzStorageTable -Name $TableName -Context $ctx
+if ($storageTable -eq $null)
+{
+    throw "Retrieving Azure storage table '$TableName' for Azure storage account '$StorageAccountName' in resource group '$ResourceGroupName' failed."
+}
+$cloudTable = ($storageTable).CloudTable
+Write-Verbose "Azure storage table '$TableName' has been retrieved for Azure storage account '$StorageAccountName' in resource group '$ResourceGroupName'"
+
+# Delete all existing entities 
+Write-Host "Deleting all existing entities in Azure storage table '$TableName' for Azure storage account '$StorageAccountName' in resource group '$ResourceGroupName'..."
+$entitiesToDelete = Get-AzTableRow -table $cloudTable
+$deletedEntities = $entitiesToDelete | Remove-AzTableRow -table $cloudTable
+Write-Host "Successfully deleted all existing entities in Azure storage table '$TableName' for Azure storage account '$StorageAccountName' in resource group '$ResourceGroupName'"
+
+# Create all new entities specified in json file
+$configFile = Get-Content -Path $ConfigurationFile | ConvertFrom-Json
+foreach ($entityToAdd in $configFile) {
+    # Check if PartitionKey provided
+    if ($entityToAdd.PartitionKey -ne $null) {
+        $partitionKey = $entityToAdd.PartitionKey
+        $entityToAdd.PSObject.Properties.Remove('PartitionKey')
+    } else {
+        $partitionKey = New-Guid
+    }
+
+    # Check if RowKey provided
```

Comment:
```suggestion
    # Check if RowKey provided
```
Not sure if this comment adds value as the code shows it.

Reply:
Removed comments, see [`cc0f8b0` (#222)](https://github.com/arcus-azure/arcus.scripting/pull/222/commits/cc0f8b0c694e07c2a87f22feecdabbf7524ab660)